lib/fluent/plugin/out_memcached.rb

```diff
@@ -13,15 +17,22 @@ class Fluent::Plugin::MemcachedOutput < Fluent::Plugin::Output
   config_param :value_format, :string, :default => 'raw'
   config_param :param_names, :string, :default => nil # nil doesn't allowed for json
 
+  config_section :buffer do
+    config_set_default :@type, DEFAULT_BUFFER_TYPE
+    config_set_default :chunk_keys, ['tag']
```

Comment:
This is for default value for chunk_keys.
Unless this value is not set, tag separated chunk does not work.